client/src/pages/_app.tsx

```diff
@@ -2,8 +2,8 @@ import * as React from 'react';
 import App from 'next/app';
 import Head from 'next/head';
 
-import '../styles/main.css';
 import 'antd/dist/antd.css';
+import '../styles/main.css';
```

Comment:
Custom styles moved to the bottom to overwrite `antd` ones (in case of same selector weights).